docs/proposals/assets/multiple-dns-provider-support/google/google.md

```diff
@@ -0,0 +1,107 @@
+## Google
+
+https://console.cloud.google.com/net-services/dns/zones
+
+Google supports Weighted(Weighted Round Robin) and Geolocation routing policies https://cloud.google.com/dns/docs/zones/manage-routing-policies. Both of these can be configured directly on records in Google Cloud DNS and no secondary Traffic Management resource is required.
+
+Notes:
+
+* Record sets are modelled as a single endpoint with routing policy embedded. This is a different approach to Route53 where each individual A/CNAME would have its own record entry. 
+* Weight must be an integer between 0 - 10000
+* There are no continent options for region, only finer grained regions such as us-east1, europe-west-1 etc...
+* There appears to be no way to set a default region, google just routes requests to the nearest supported region.
```

Comment:
hmm so this prob affects our proposed API for DNSPolicy? no point asking for a default if only one provider supports it?

Reply:
I think Azure might have an option:

![image](https://github.com/Kuadrant/multicluster-gateway-controller/assets/327352/e18db8c0-a1ac-44fa-ae63-d3cd483a36d1)

I'm assuming that sets the NA region as the default, but EU traffic should go to the other one. Need to verify that.